src/garage/tf/embeddings/utils.py

```diff
@@ -0,0 +1,23 @@
+"""Utilities for embedding."""
```

Comment:
this should probably be an akro features instead? does akro have this?

Reply:
I removed this. I will add this back in a PR when it is actually needed.